frontend/src/components/custom-tools/combined-output/CombinedOutput.jsx

```diff
@@ -96,14 +99,23 @@ function CombinedOutput({ docId, setFilledFields }) {
   }, [combinedOutput]);
 
   const handleOutputApiRequest = async () => {
-    const requestOptions = {
+    const requestPrivateOptions = {
       method: "GET",
       url: `/api/v1/unstract/${sessionDetails?.orgId}/prompt-studio/prompt-output/?tool_id=${details?.tool_id}&document_manager=${docId}&is_single_pass_extract=${singlePassExtractMode}`,
       headers: {
         "X-CSRFToken": sessionDetails?.csrfToken,
       },
     };
-
+    const requestPublicOptions = {
+      method: "GET",
+      url: `/public/share/outputs-metadata/?id=${id}&document_manager=${docId}&is_single_pass_extract=${singlePassExtractMode}`,
```

Comment:
@harini-venkataraman Can we get the API from cloud repo? Please create a helper function that returns the above API. Please refer to [this](https://github.com/Zipstack/unstract/blob/83809f77efb90a0901877e02caaaf8db543dda71/frontend/src/components/custom-tools/combined-output/CombinedOutput.jsx#L108).